helix-view/src/tree.rs

```diff
@@ -360,7 +366,27 @@ impl Tree {
             return;
         }
 
-        self.stack.push((self.root, self.area));
+        let area = if self.max_width > 0 {
```

Comment:
Is it really useful behavior for non-zoom state? I would expect this option to affect only zoomed window.

Reply:
I believe it is, since it's a fairly distinct feature and could have possible uses outside just in a zoomed mode, such as for a wide-screen, or for centring the editor with vertical splits. It's not guaranteed that this will be set when zoomed, either, so it's not always going to be used. It also does not have a config option (they've been removed from this PR now), so it won't ever be set when helix starts anyway.

My main feeling with it, though, is that if it was made zoom-only, then the question would start being asked "why not when not zoomed?" and I wouldn't have a good answer. Having it work outside the zoom makes the feature more flexible, which in turn makes it more useful.